### PR TITLE
Added links to the go template docs on golang site

### PR DESCRIPTION
**Pull request by letzya**

### CODE REVIEW

**Comment by letzya** [comment]
@asoorm can you approve please

File: tyk-docs/content/transform-traffic/request-body.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ weight: 3
 
 Sometimes you may be exposing an older API, or one that uses a legacy structure for input data, or you are actually creating a new API schema and models that are cleaner which you want to apply to your existing API without modifying it (it may have many legacy clients that cannot be upgraded).
 
+For body transform we are using golang template. See [GoLang doc to learn how to use it](https://golang.org/pkg/html/template/) and [a blog about it](https://blog.gopheracademy.com/advent-2017/using-go-templates/).
```
Comment (joshblakeley) [inline tyk-docs/content/transform-traffic/request-body.md]: Just personal pref but some suggested grammar changes:

Our body transform middleware uses the Go template language. See [Godoc to learn more](https://golang.org/pkg/text/template/) and [a useful blogpost on using Go templates](https://blog.gopheracademy.com/advent-2017/using-go-templates/).

Comment (buger) [reply tyk-docs/content/transform-traffic/request-body.md]: @marksou link should point to https://golang.org/pkg/text/template/ instead

Comment (asoorm) [reply tyk-docs/content/transform-traffic/request-body.md]: I like @joshblakeley suggestion. Grammar fix & correct links

Comment (letzya) [reply tyk-docs/content/transform-traffic/request-body.md]: done. @asoorm

**Review by asoorm — APPROVED** [review]

**Pull request merged**